src/main/java/com/conveyal/gtfs/util/GeoJsonUtil.java

```diff
@@ -411,15 +394,14 @@ private static List<Point> buildPoints(List<LocationShape> locationShapes) {
     /**
      * From the provided list of {@link LocationShape}s extract all line string entries and group by line string id.
      */
-    private static List<LocationShape> getLineStings(
-        String locationId,
-        List<LocationShape> locationShapes
-    ) {
+    private static List<LocationShape> getLineStings(String locationId, List<LocationShape> locationShapes) {
```

Comment:
Typo "stings"....

Reply:
Fixed in https://github.com/ibi-group/gtfs-lib/pull/7/commits/43bd72f5a3d0ddd8da9061dbc57f9e88cecf972e